app/src/main/java/se/inera/intyg/minaintyg/config/WebSecurityConfig.java

```diff
@@ -182,4 +185,13 @@ private String getAttribute(Saml2Authentication samlCredential) {
     throw new IllegalArgumentException(
         "Could not extract attribute '" + PERSON_ID_ATTRIBUTE + "' from Saml2Authentication.");
   }
+
+  @Bean
+  public FilterRegistrationBean<SessionTimeoutFilter> sessionTimeoutFilter() {
```

Comment:
I had issues starting the application, solution was to set the filter like this:
```java
 @Bean
  public FilterRegistrationBean<SessionTimeoutFilter> sessionTimeoutFilter() {
    final var filterRegistrationBean = new FilterRegistrationBean<SessionTimeoutFilter>();
    filterRegistrationBean.setFilter(new SessionTimeoutFilter(sessionTimeoutService));
    filterRegistrationBean.addInitParameter("skipRenewSessionUrls", SESSION_STATUS_CHECK_URI);
    filterRegistrationBean.addUrlPatterns("/*");

    return filterRegistrationBean;
  }
```

Reply:
This part isn't finished yet, it is there temporary so @mww-sopra-steria has time to decide how and where it was supposed to be added